darwin/importer/importer.py

```diff
@@ -21,7 +25,7 @@ def build_main_annotations_lookup_table(annotation_classes):
 def find_and_parse(
     importer: Callable[[Path], Union[List[dt.AnnotationFile], dt.AnnotationFile, None]],
     file_paths: List[Union[str, Path]],
-) -> (List[dt.AnnotationFile], List[dt.AnnotationFile]):
+) -> Tuple[List[dt.AnnotationFile], List[dt.AnnotationFile]]:
     # TODO: this could be done in parallel
```

Comment:
Should we create tickets for TODO tags?
How can we prevent this codebase from being filled with TODO/FIXME tags? How are we addressing them?

Reply:
Good point, done: https://linear.app/v7labs/issue/V7-1221/parsing-files-in-importer-can-be-done-in-parallel